cherrypy/test/test_logging.py

```diff
@@ -199,7 +199,8 @@ def test_custom_log_format(log_tracker, monkeypatch, server):
 
 
 def test_utc_in_timez(monkeypatch):
-    """Test that ``LazyRfc3339UtcTime`` is rendered as ``str`` using UTC timestamp."""
+    """Test that ``LazyRfc3339UtcTime``
+    is rendered as ``str`` using UTC timestamp."""
```

Comment:
This seems broken PEP 257-wise. Any ideas of how to make it look better? The title line is usually a full sentence on a single line. Max 72 chars. Then, there's an empty line and a paragraph of longer text if needed.

Reply:
I think I shortened the sentence to fit 257. Just pushed a patch.

Reply:
It looks like you accidentally marked this as resolved — there's no new commit. OTOH, it's probably best to separate it out. In #2007 you also modify this line in a different way. It'd be easier to accept it as a separate patch that doesn't influence both of the unrelated PRs.